examples/test/main.ml

```diff
@@ -198,24 +196,11 @@ let () =
 
 (* Custom mapping between association lists and JS objects *)
 
-module Dict : sig
+module Dict = [%js:
```

Comment:
IIUC, this completely changes the behavior of Dict.t.  The assoc list is no longer represented as a JS object.